app/lucy/src/app/components/Utilities/map-preview/map-preview.component.ts

```diff
@@ -319,28 +320,25 @@ export class MapPreviewComponent implements OnInit, AfterViewInit, AfterViewChec
       }
     })
     .bindTooltip(function (feature) {
-      return `${feature.feature.properties.DISTRICT_NAME}`;
+      return `${feature.feature.properties.ADMIN_AREA_NAME}`;
     }).addTo(regionalDistrictsLayerGroup);
     regionalDistrictsLayerGroup.addTo(this.map);
   }
 
   private async addWellsLayerToMap(bbox: number[]) {
+    const wellIcon = L.icon({
+      iconUrl: encodeURI('data:image/svg+xml,' + WaterDropletSVG.waterDroplet),
```

Comment:
Just curious, I don't know how painful it is to configure the leaflet. But did you try storing the svg in some folder with .svg extension and linking it by just providing the path here?

Reply:
I did try - I couldn't get it to import the SVG file. A bug in Leaflet related to this was reported a few years ago; it looks like it hasn't been resolved yet.

Reply:
uh-oh! alright then 👍